File: Python/0.1-Basics/0.3-Conditional-Statements-Advanced/Exercise/cinema.py
projection_type = input()
rows = int(input())
columns = int(input())
total_seats = rows * columns
price = 0

if projection_type == "Premiere":
    price = 12
elif projection_type == "Normal":
    price = 7.50
elif projection_type == "Discount":
    price = 5

income = price * total_seats
print(f"{income:.2f}")

File: JS-Basics/0-3-Conditional-Steatements-Advanced/Exercise/cinema.js
function cinema(input) {
    let projectionType = input[0];
    let rows = Number(input[1]);
    let columns = Number(input[2]);
    let ticketPrice;

    if (projectionType == "Premiere") {
        ticketPrice = 12;
    } else if (projectionType == "Normal") {
        ticketPrice = 7.50;
    } else if (projectionType == "Discount") {
        ticketPrice = 5;
    }

    ticketPrice = ticketPrice * (rows * columns);
    console.log(ticketPrice.toFixed(2))
}

cinema(["Premiere", "10", "12"])
cinema(["Normal", "21", "13"])
cinema(["Discount", "12", "30"])
